gemfiles/Gemfile.rails6

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'rails', '~> 6'
+
+gemspec path: "..", name: 'nylas-streaming'
```

Comment:
I know why this is here but perhaps it might be a good idea to add a comment as to why this gemspec is included? What do you think?